src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/GroupBurdenEstimatesController.kt

```diff
@@ -84,11 +127,9 @@ open class GroupBurdenEstimatesController(context: ControllerContext) : Abstract
         return objectCreation(context, url)
     }
 
-    private fun getValidResponsibilityPath(
-            context: ActionContext,
-            estimateRepository: BurdenEstimateRepository,
-            readEstimatesRequired: Boolean = false
-    ): ResponsibilityPath
+    private fun getValidResponsibilityPath(context: ActionContext,
+                                           estimateRepository: BurdenEstimateRepository,
+                                           readEstimatesRequired: Boolean = false): ResponsibilityPath
```

Comment:
Is this a preference for laying out function signatures, or just an accidental refactor? Because if it's just an accident, I prefer the former :)

Reply:
This was not intentional, happy to revert :)